src/test/scala/chiselTests/experimental/ForceNames.scala

```diff
@@ -0,0 +1,108 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import firrtl._
+import chisel3._
+import chisel3.core.annotate
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage}
+import chisel3.util.experimental.{ForceNameAnnotation, ForceNamesTransform, InlineInstance, forceName}
+import firrtl.annotations.{Annotation, ReferenceTarget}
+import firrtl.options.{Dependency, TargetDirAnnotation}
+import firrtl.stage.RunFirrtlTransformAnnotation
+import logger.{LogLevel, LogLevelAnnotation}
+
+/** Object containing Modules used for testing */
+object ForceNamesHierarchy {
+  class WrapperExample extends MultiIOModule {
```

Comment:
Similar to Chick's comment above, would prefer names in line with Scala-style below.